src/main/java/ch/njol/skript/util/SkriptColor.java

```diff
@@ -182,6 +182,15 @@ public static SkriptColor fromBukkitColor(org.bukkit.Color color) {
 		assert false;
 		return null;
 	}
+
+	public static SkriptColor fromChatColor(ChatColor chatColor) {
+		for (SkriptColor color : colors) {
+			if (color.chat == chatColor)
+				return color;
+		}
+		assert false;
```

Comment:
This assert should not be here. The assert exists in the other methods because they're never going to return null, a value for them is always defined.

In this case. SkriptColor doesn't implement all the ChatColor enums as Spigot also throws the Chat styles into this enum. So this needs to be removed or else if someone casts ChatColor.BOLD through the method, it will error out.